Priyaa/HTML Basics Remake/Assignment-3.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<h1><b>Register</b></h1>
+<form>
+<b>First name:</b>
+<input type="text" name="firstname" placeholder="First Name">
+<b>Last name:</b>
+<input type="text" name="lastname" placeholder="Last Name">
+<br><br>
+     
+     <b>Gender:</b>
+     <input type="radio" name="gender" value="male" checked> Male
+     <input type="radio" name="gender" value="female"> Female
+     <input type="radio" name="gender" value="other"> Others
```

Comment:
`required` attribute missing. And male shouldn't be checked by default as it can end up in accidental submission and also offensive to other genders in certain cases. `label` needs to be used.